### DISCUSSION TITLE
POLL - What platform are you using deployKF on?

### DISCUSSION THREAD

**Discussion by thesuperzapper** [deployKF/deployKF#84] [Polls]
This poll helps us understand what platforms are most popular with deployKF users, and helps us prioritize what features we add next!

**Comment by pbasov** [comment]
Baremetal kubernetes cluster, using calico, ceph storage.

**Reply by quloos** [reply]
Same

**Comment by bobbeeke** [comment]
Self hosted Kubernetes cluster on AWS, deployed with Kops, Ubuntu LTS node AMIs.
S3 (AWS cloud) object store for pipelines.
Self hosted (in cluster) mysql databases for Katib and ml-pipelines 
Nginx ingress in front of deploykf gateway
Self hosted (in cluster) cert-manager, TLS, Lets-encrypt.
Route53 DNS

Still use ArgoCD plugin mode and sync script for rollout but may use the manifest repo mode in the future.

**Comment by haiminh2001** [comment]
K3s cluster with flannel.

**Comment by ReggieCarey** [comment]
Baremetal, Proxmox, Rocky Linux 9 VM, Rancher RKE2.
